src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -667,6 +667,22 @@ public async Task CloseAsync_ByServer_AcceptThrows()
             }
         }
 
+        [Fact]
+        public async Task CloseAsync_MultipleCalls_FollowingCallsAreIgnored()
```

Comment:
Can we also add a test with multiple server-side `CloseAsync`?

Reply:
done